pkg/offline/install.tmpl

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
```

Comment:
Mention that a conda environment with python=3.7 and a pip that supports `--use-feature=2020-resolver` must be present on the target machine.